src/core-services/account/mutations/index.js

```diff
@@ -11,6 +11,7 @@ import sendResetAccountPasswordEmail from "./sendResetAccountPasswordEmail.js";
 import setAccountProfileCurrency from "./setAccountProfileCurrency.js";
 import setAccountProfileLanguage from "./setAccountProfileLanguage.js";
 import updateAccountAddressBookEntry from "./updateAccountAddressBookEntry.js";
+import createGroup from "./createAccount"
```

Comment:
I think your filename is wrong. Should it be `createGroup`?

Reply:
yeah it should be `createGroup `...  good catch